app/workers/organizations/update_organization_articles_paths_worker.rb

```diff
@@ -0,0 +1,15 @@
+module Organizations
+  class UpdateOrganizationArticlesPathsWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority
+
+    def perform(organization_id, old_slug, new_slug)
+      organization = Organization.find_by(id: organization_id)
+
+      return unless organization
+
+      organization.articles.find_each { |article| article.update(path: article.path.gsub(old_slug, new_slug)) }
```

Comment:
this could be split in multiple lines using do 😊

Reply:
Definitely :)
Also, wouldn't this worker be in the `workers/articles` folder instead? @rhymes

Reply:
Mmm I think either is fine to be honest, your call :) The worker is receiveing an organization as its argument but then acting on their articles 🤷‍♂️